### ISSUE TITLE
Original image leaking during reconstruction

### ISSUE THREAD

**Issue by zhanghm1995** [MCG-NJU/VideoMAE#29]
Hi,
thanks for you sharing this code. 

I have one question that you use the nn.Conv3d operation in [L145](https://github.com/MCG-NJU/VideoMAE/blob/e93f386cffee601cbb01882024255a1dbc23128f/modeling_finetune.py#L145) in `PatchEmbed` class to mitigate the patching process, which gets the patches in temporal and spatial dimensions. After this operation, you use `x_vis = x[~mask].reshape(B, -1, C) # ~mask means visible` to get the unmasked patches.

However, I am curious about whether this learnable nn.Conv3d module could extract the original masked image features, and resulting in the masked patches information leaking? Since the respective field of this 3D convolution may contain the masked patches in the original images.

I hope you could solve my confusion.
